CHANGELOG.md

```diff
@@ -9,9 +9,366 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+
+### Configuration changes
+
+-   Added `use_dynamic_access_token_signing_key` (defaults to `True`) option to the Session recipe config
+-   Added `expose_access_token_to_frontend_in_cookie_based_auth` (defaults to `False`) option to the Session recipe config
+-   JWT and OpenId related configuration has been removed from the Session recipe config. If necessary, they can be added by initializing the OpenId recipe before the Session recipe.
+
+
+### Interface changes
+
 - Renamed `get_session_data` to `get_session_data_from_database` to clarity that it always hits the DB
 - Renamed `update_session_data` to `update_session_data_in_database`
 - Renamed `session_data` to `session_data_in_database` in `SessionInformation` and the input to `create_new_session`
+- Added new `check_database` param to `verify_session` and `get_session`
+- Removed `status` from `jwks_get` output (function & API)
+- Added new optional `use_static_signing_key` param to `createJWT`
+- Removed deprecated `update_access_token_payload` and `regenerate_access_token` from the Session recipe interface
+- Removed `get_access_token_lifetime_ms` and `get_refresh_token_lifetime_ms` functions
+
+
+## Changes
+
+-   The Session recipe now always initializes the OpenID recipe if it hasn't been initialized.
+-   Refactored how access token validation is done
+-   Removed the handshake call to improve start-up times
+-   Added support for new access token version
+
+### Added
+
+-   Added `create_new_session_without_request_response`, `get_session_without_request_response`, `refresh_session_without_request_response` to the Session recipe.
+-   Added `get_all_session_tokens_dangerously` to session objects (`SessionContainer`)
+-   Added `attach_to_request_response` to session objects (`SessionContainer`)
+
+### Migration
+
+#### If self-hosting core
+
+1. You need to update the core version
+2. There are manual migration steps needed. Check out the core changelogs for more details.
+
+#### If you used the jwt feature of the session recipe
+
+1. Add `expose_access_token_to_frontend_in_cookie_based_auth=true` to the Session recipe config on the backend if you need to access the JWT on the frontend.
+2. On the frontend where you accessed the JWT before by: `(await Session.getAccessTokenPayloadSecurely()).jwt` update to:
+
+```tsx
+let jwt = null;
+const accessTokenPayload = await Session.getAccessTokenPayloadSecurely();
+if (accessTokenPayload.jwt === undefined) {
+    jwt = await Session.getAccessToken();
+} else {
+    // This branch is only required if there are valid access tokens created before the update
+    // It can be removed after the validity period ends
+    jwt = accessTokenPayload.jwt;
+}
+```
+
+3. On the backend if you accessed the JWT before by `session.get_access_token_payload()['jwt']` please update to:
+
+```python
+jwt = None
+access_token_payload = await session.get_access_token_payload_securely() # FIXME
```

Comment:
FIXME

Reply:
I've updated the changelog in the node base branch; you can fix these.

Reply:
I still seem them in Node SDK changelog but nowhere in the SDK code.

Reply:
Fixed. There was a seperate PR (branch)